### PR TITLE
Adds Vincent's Hacktoberfest story

### PR DESCRIPTION
**Pull request by simonprickett**

### CODE REVIEW

**Comment by simonprickett** [comment]
@SuzeShardlow I'll review this locally and comment here, but you'll have to approve it if my review is good as I can't do an official review on a PR I created oops.

File: docs/get-involved/hacktoberfest/vincent-aceto-story/index-vincent-aceto-story.mdx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+Now, rewind the clock a bit, and you'd find me perusing Github's Hacktoberfest-tagged issues.  I wanted to find the best first contribution for the month's coding festivities.  While searching, I had one very important criterion that the introductory issue needed to satisfy: to work with a technology that I do not use on a daily basis.  I wanted to make sure that I walked away with a newfound knowledge that would benefit my career.  After some time, my eyes landed on a Redis Developer Community's issue - I knew it immediately, this was perfect!  The checkbox was ticked, for I do not regularly work with Redis.  I was now ready to kick off the Hacktoberfest celebration.
```
Comment (simonprickett) [inline docs/get-involved/hacktoberfest/vincent-aceto-story/index-vincent-aceto-story.mdx]: @SuzeShardlow suggest changing "landed on a Redis Developer Community's issue" to "landed on a Redis Developer Community isue"

File: docs/get-involved/hacktoberfest/vincent-aceto-story/index-vincent-aceto-story.mdx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
...](https://github.com/redis-developer/introducing-the-geosearch-command/issues/5).  The issue's aim was to extend an existing Docker Compose integration.  The `docker-compose.yml` file was to include a schema, which was to run the repository's Python Flask application in a Docker container.  Additionally, the main application was to connect to the project's existing Redis container - this Redis build step was already included in the Docker Compose file.  With the features and constraints clearly defined, the next step was to pull out the documentation.  To make sure I was familiar with the tech at hand, and to ensure I got the most out of working on the issue, I started with the Redis installation docs - becoming aware that things like the default Redis `port 6379` would come to serve me well when debugging.  After installation, I took some time to explore the Redis API and read about Redis' internal hash implementation at a high level.  The final reconnaissance was to review Docker.  I had briefly used Docker at a previous position, and have worked on some personal projects using the container technology; however, a quick Dockerfile and `docker-compose.yml` refresher was necessary.
```
Comment (simonprickett) [inline docs/get-involved/hacktoberfest/vincent-aceto-story/index-vincent-aceto-story.mdx]: @SuzeShardlow suggest removing the code font from `port 6379` and just making the code font on the `6379`.

**Review by simonprickett — COMMENTED** [review]
Left a couple of comments.

File: docs/get-involved/hacktoberfest/vincent-aceto-story/index-vincent-aceto-story.mdx (hunk 1 above)
Comment (simonprickett) [reply docs/get-involved/hacktoberfest/vincent-aceto-story/index-vincent-aceto-story.mdx]: Fixed in 0eb43d22d

File: docs/get-involved/hacktoberfest/vincent-aceto-story/index-vincent-aceto-story.mdx (hunk 2 above)
Comment (simonprickett) [reply docs/get-involved/hacktoberfest/vincent-aceto-story/index-vincent-aceto-story.mdx]: Fixed in 0eb43d22d

**Review by SuzeShardlow — APPROVED** [review]
Look good to me!

**Pull request merged**